code/code/misc/combat.cc

```diff
@@ -2190,7 +2190,7 @@ int TBeing::hit(TBeing *target, int pulse)
      !heldInPrimHand() && !heldInSecHand() &&
      doesKnowSkill(SKILL_BLUR) && 
      ::number(0, 99) < 20 &&  // this makes it happen 20% of the time
-     !isAffected(AFF_ENGAGER) && getMana()>=25 &&
+     !isAffected(AFF_ENGAGER) && getMana()>=15 &&
```

Comment:
This is just a check like "do have have enough many to pay for this if we do it?"
You'll need to update the reconcileMana() call as well in order for this to work. It might break things this way.